sql/telemetry_derived/ssl_ratios_v1/metadata.yaml

```diff
@@ -6,7 +6,8 @@ owners:
   - chutten@mozilla.com
 labels:
   application: firefox
-  incremental: true
+  incremental: false
```

Comment:
`incremental` is a bit confusing here since the query itself is incremental (it produces new data every day and appends it to the table), but it should be treated as non-incremental. That probably needs some further discussion when we get to implementing low-friction scheduled queries.

Reply:
We should definitely note this gotcha as a comment here.

Reply:
This should get added into the README at https://github.com/mozilla/bigquery-etl#query-metadata.

Reply:
Hmm. Let's open an issue for how we want to solve this. I believe that we want a separate label for how the data export is handled; e.g.
```
incremental: true
incremental_export: false
```